pkg/build/builder/cmd/dockercfg/cfg.go

```diff
@@ -41,7 +41,10 @@ func (h *Helper) InstallFlags(flags *pflag.FlagSet) {
 
 // GetDockerAuth returns a valid Docker AuthConfiguration entry, and whether it was read
 // from the local dockercfg file
-func (h *Helper) GetDockerAuth(imageName, authType string) (docker.AuthConfiguration, bool) {
+func (h *Helper) GetDockerAuth(imageName, runtimeImageName, authType string) (docker.AuthConfiguration, bool) {
+	if runtimeImageName != "" {
+		glog.V(3).Infof("Runtime image detected %s", runtimeImageName)
+	}
```

Comment:
But maybe we could put it inside of `if s.build.Spec.Strategy.SourceStrategy.RuntimeImage != nil {` ? And I see that the similar message already here, so I'm not sure should we show to the user `runtime image detected` message twice.